cmd/lakectl/cmd/local.go

```diff
@@ -146,6 +147,25 @@ func localHandleSyncInterrupt(ctx context.Context) context.Context {
 	return ctx
 }
 
+func dieOnInterruptedOperation(interruptedOperation string, force bool) {
+	if !force {
+		switch strings.ToLower(interruptedOperation) {
```

Comment:
Consider passing _the death message_ as a parameter to this func.  It is easiest to review the message at the call site of dieOnInterruptedOperation -- and that way you don't need a default case here.

Reply:
The issue is that the death message depends on the latest failed operation and not the current operation so in case the current operation is `local pull`, for example, I'll still need to render a different death message for each latest failed operation that happened